styles/desktop-styles.css

```diff
@@ -0,0 +1,284 @@
+@media screen and (min-width: 769px) {
+  main {
+    background-image: url(../assets/header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .header {
+    display: none;
+  }
+
+  .nav-desktop {
+    display: flex;
+    justify-content: center;
+  }
+
+  .works {
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+  }
+
+  .project-grid {
+    display: grid;
+    align-self: center;
+    justify-self: center;
+    grid-template-columns: repeat(auto-fit, minmax(14.8rem, 1fr));
+    grid-template-rows: repeat(auto-fit, minmax(auto, 1fr));
+    border-radius: 8px;
+    padding: 10px;
+    row-gap: 10px;
+    column-gap: 8px;
+  }
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement as I mentioned above in my inline comment and even mentioned by your previous code reviewer regarding your design not matching properly with the Figma design. Your design should start responsive from the `768px` screen size. Right now because as I can see you are not using the Correct `layout`, `media-query` and etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-17 04-31-43](https://github.com/BHObserver/portfolio/assets/66476329/4eaf08dc-cbb8-4086-b476-b92f373818a3)

</td>
<td>

![0268dd30-a845-4a5a-b262-c57e5f68cc18](https://github.com/BHObserver/portfolio/assets/66476329/c93e1fbc-6c95-4534-86d6-d2213f336dc5)

</td>
</tr>
</table>


**Hint**: *Please reduce the card width and height manually inside the desktop media query >=768px and please use the `grid` property properly.*